### DISCUSSION TITLE
请问如何指定特定网站使用第二DNS解析？

### DISCUSSION THREAD

**Discussion by EliteOtaku** [vernesong/OpenClash#3698] [General]
目前我有2个网站不知道为何一直无法正常解析
www.wstx.com
![image](https://github.com/vernesong/OpenClash/assets/29235152/f4b4ab9f-97d6-4dcd-a7ae-09d23c264eb3)
www.expreview.com
![image](https://github.com/vernesong/OpenClash/assets/29235152/8f7a5917-fb05-4397-adf3-0a7c62467073)

我现在设置如下，请问是否正确？
![image](https://github.com/vernesong/OpenClash/assets/29235152/fb454c7a-220b-493a-841a-4a298a3b0f9a)

**Comment by vernesong** [comment]
去掉井号
